pyproject.toml

```diff
@@ -25,6 +25,7 @@ dependencies = [
     "pydantic>=2",
     "pysnmplib",
     "pyasn1<0.5.0",
+    "tomli",
```

Comment:
Only needed for python < 3.11, python >= 3.11 comes with tomllib

Reply:
This can be fixed with a modif

I guess this can be resolved with a `tomli  ; python_version < '3.11'` as a replacement, but I would assume an import guard is needed in the code as well, sinece the modules are named differently...